arch/arm64/boot/dts/broadcom/bcm283x-rpi-csi1-2lane.dtsi

```diff
@@ -0,0 +1 @@
+../../../../arm/boot/dts/bcm283x-rpi-csi1-2lane.dtsi
```

Comment:
Just a hint, the CM3 will be added in 4.20 so the other csi variants could be added too

Reply:
They could, but creating files that are unused and untested tends to cause more upset than having to link to them later on.

Reply:
Okay